packages/saber-plugin-query-posts/README.md

```diff
@@ -101,6 +124,35 @@ hello
 
 Then there will be a `/categories/sports` page，`categories` is an array, so you can assign the post to multiple categories.
 
+### Data Injected in Post Pages
+
+You can access the detailed info of the categories of a post page via `this.page.categories`, for instance:
+
+```yaml
+categories:
+  - sports/football
+  - hobby
+```
+
+gives you:
+
+```js
+;[
+  {
+    name: 'sports',
+    permalink: '/categories/sports'
+  },
+  {
+    name: 'sports/fooball',
+    permalink: '/categories/sports/fooball'
+  },
+  {
```

Comment:
Another thing I'm not entirely sure, should be `name` be `football` or `sports/football`?